src/Pie.js

```diff
@@ -79,10 +87,10 @@ export default class Pie extends Viz {
         y: 0
       })
       .config(configPrep.bind(this)(this._shapeConfig, "shape", "Path"))
+      .config({ariaLabel: constructAriaLabel.bind(this)})
```

Comment:
we want the user to be able to override this default behavior, so please move all instances of the named `constructAriaLabel` function directly into the shapeConfig in the constructor.

Reply:
@rbaheti you only half-understood my comment. This needs to live inside of the constructor of the class (see [this](https://github.com/d3plus/d3plus-hierarchy/pull/38/files#diff-695c6459e25c907f953d398535f788baR25) line). Currently, by setting this after the user's defined config (which is what `this._shapeConfig` is), we are overwriting any user-defined `ariaLabel` (which we do not want to do).